### ISSUE TITLE
Suggestion for the Arc OSX Darker menu color

### ISSUE THREAD

**Issue by antonioidalecio** [LinxGem33/OSX-Arc-Darker#15]
Could you add a dark color to Arc OSX Darker theme menu just like the Arc OSX Shadow menu color? I think this would be much better than the white one!!

**Comment by mesolaries** [comment]
Yes, please, add shadow menu color to darker theme or change button colors to dark. Otherwise the buttons are not visible at all.
